specification/eventgrid/Azure.Messaging.EventGrid/main.tsp

```diff
@@ -114,34 +115,31 @@ namespace Microsoft.EventGrid {
     value: ReceiveDetails[];
   }
 
-  @doc("Failed lock token information.")
+  @doc("Failed LockToken information.")
   model FailedLockToken {
-    @doc("The lock token of an entry in the request.")
+    @doc("LockToken value")
     lockToken: string;
 
-    @doc("Error code related to the lock token. Example of such error codes are TokenLost: which indicates that token is not found, and InternalServerError: For any internal server errors.")
-    errorCode: string;
-
-    @doc("Description of the lock token error.")
-    errorDescription: string;
+    @doc("Error information of the failed operation result for the lock token in the request.")
+    error: Azure.Core.Foundations.Error;
   }
 
   @doc("Supported delays for release operation.")
-  enum ReleaseDelay {
+  enum Delay {
```

Comment:
Not blocking, but this will be too general for an SDK. `ReleaseDelay` was more specific and less likely to conflict.